app/uk/gov/hmrc/cdsimportsddsfrontend/services/xml/XmlWriter.scala

```diff
@@ -84,11 +84,21 @@ object XmlWriterInstances {
   }
 
   def maybeElement(elementName: String, maybeElementValue: Option[String]): Option[Node] = {
-    maybeElementValue.filter(_.nonEmpty).map(value => element(elementName, value))
+    maybeElementValue.filter(_.nonEmpty).map(value => element(elementName, value, None))
   }
 
-  private def element(elementName: String, elementValue: String): Node = {
-    Elem.apply(null, elementName, scala.xml.Null, scala.xml.TopScope, true, Text(elementValue)) //scalastyle:ignore
+  def maybeElement(elementName: String, maybeElementValue: Option[String], attribute: Option[Attribute]): Option[Node] = {
```

Comment:
could this just be `def maybeElement(elementName: String, maybeElementValue: Option[String], attribute: Option[Attribute] = None)` ?

Reply:
:+1: 
Fixed